plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/server/debug/OpenShiftDebugUtils.java

```diff
@@ -155,36 +171,99 @@ public void done(IJobChangeEvent event) {
 
 	private void updateDeploymentConfigValues(IDeploymentConfig deploymentConfig,
 			DebuggingContext debugContext) {
```

Comment:
this method is pretty big. can you split each branch of the if condition into 2 simpler methods?